docs/ops/movement/Gather_7.md

```diff
@@ -0,0 +1,167 @@
+## Gather <a name="Gather"></a> {#openvino_docs_ops_movement_Gather_7}
+
+**Versioned name**: *Gather-7*
+
+**Category**: Data movement operations
+
+**Short description**: *Gather* operation takes slices of data in the first input tensor according to the indices
+ specified in the second input tensor and axis from the third input. Semantic of this operation is identical to 
+*Tensorflow* GatherV2.
```

Comment:
Change to `TensorFlow\* [Gather](https://www.tensorflow.org/api_docs/python/tf/gather) operation.`

Reply:
But how about case when `batch_dims < 0`? Our specification does not allow negative values for this attribute.

Reply:
1. will add such link
2. about `batch_dims < 0` answered in email

Reply:
changed reference to TF Gather